src/odemis/driver/orsay.py

```diff
@@ -148,6 +164,33 @@ def __init__(self, name, role, children, host, daemon=None, **kwargs):
             self._gis_reservoir = GISReservoir(parent=self, daemon=daemon, **kwargs)
             self.children.value.add(self._gis_reservoir)
 
+        # create the FIB device child
+        try:
+            kwargs = children["fib-device"]
+        except (KeyError, TypeError):
+            logging.info("Orsay was not given a 'fib-device' child")
+        else:
+            self._fib_device = FIBDevice(parent=self, daemon=daemon, **kwargs)
+            self.children.value.add(self._fib_device)
+
+        # create the FIB source child
+        try:
+            kwargs = children["fib-source"]
+        except (KeyError, TypeError):
+            logging.info("Orsay was not given a 'fib-source' child")
```

Comment:
I'm not that into the naming/branding of Orsay but I thought Orsay was a brand isn't it? Then maybe it's nicer to rewrite this log to:
> "The Orsay microscope was not given a 'fib-source' child"

If you think that makes any sense.

Reply:
That does make sense, I've updated it to 'The Orsay component'...